app/models/concerns/bike_searchable.rb

```diff
@@ -14,8 +14,14 @@ module ClassMethods
     #   distance: distance in miles for matches. Only for stolenness == 'proximity'
     #   bounding_box: bounding box generated by geocoder. Only for stolenness == 'proximity'
     def searchable_interpreted_params(query_params, ip:)
-      serial = SerialNormalizer.new(serial: query_params[:serial]) if query_params[:serial].present?
-      (serial ? { serial: serial.normalized } : {}) # serial if present
+      params = {}
```

Comment:
Why is this here?

Reply:
aka, why not add this stuff to `query_params`?

Reply:
Mainly following suit with the original code, which builds a new hash. 
I see merit in that approach though, since mutating a function parameter feel like an act of hostility.